src/core.js

```diff
@@ -825,6 +825,7 @@ define([
             data.node.icon = mug.getIcon();
             _this.refreshCurrentMug();
         }).bind("deselect_all.jstree deselect_node.jstree", function (e, data) {
+            $('.atwho-container').remove();
```

Comment:
This would actually cause issues if atwho was ever implemented outside of a question type, but I don't see that happening anytime soon.